lib/wkhtmltopdf_binary.rb

```diff
@@ -1,15 +1,14 @@
 class WkhtmltopdfBinary
   def self.path
     @path ||= begin
-      arch = case RUBY_PLATFORM
-        when /linux/
-          'linux-x86'
-        when /darwin/
-          'darwin-x86'
-        else
-          raise "Invalid platform. Must be running linux or intel-based Mac OS."
+      cpu, os = RUBY_PLATFORM.split('-', 2)
+      if os != 'linux' && os != 'darwin'
+        raise "Invalid platform. Must be running linux or intel-based Mac OS."
       end
-      File.expand_path "#{File.dirname(__FILE__)}/../libexec/wkhtmltopdf-#{arch}"
+      path = "#{File.dirname(__FILE__)}/../libexec/"
```

Comment:
On my Mac the `RUBY_PLATFORM` value is `"x86_64-darwin15.0"`. So the code won't work. I suggest to add tests for the code because it looks much complicated and it easy to make mistake here.